staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4876,6 +4879,9 @@ type PodStatus struct {
 	// e.g. 'Evicted'
 	// +optional
 	Reason string `json:"reason,omitempty" protobuf:"bytes,4,opt,name=reason"`
+	// The generation of the pod observed at the point this pod status is being reported.
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"varint,20,opt,name=observedGeneration"`
```

Comment:
I'm not sure if there is a protocol for how to pick the field numbers for new fields. The ones I picked are arbitrary.

Reply:
It's auto generated, but I'm not sure if it's auto-updated once set. Might be best to just remove the protobuf annotations and rerun `make update`, but @liggitt would know better than me.